nsqd/nsqd_test.go

```diff
@@ -353,17 +353,22 @@ func TestCluster(t *testing.T) {
 	equal(t, err, nil)
 
 	url := fmt.Sprintf("http://%s/topic/create?topic=%s", nsqd.RealHTTPAddr(), topicName)
-	_, err = http_api.NegotiateV1("POST", url, nil)
+	err = http_api.POSTV1(url)
 	equal(t, err, nil)
 
 	url = fmt.Sprintf("http://%s/channel/create?topic=%s&channel=ch", nsqd.RealHTTPAddr(), topicName)
-	_, err = http_api.NegotiateV1("POST", url, nil)
+	err = http_api.POSTV1(url)
 	equal(t, err, nil)
 
 	// allow some time for nsqd to push info to nsqlookupd
 	time.Sleep(350 * time.Millisecond)
 
-	data, err := http_api.NegotiateV1("GET", fmt.Sprintf("http://%s/debug", lookupd.RealHTTPAddr()), nil)
+	endpoint := fmt.Sprintf("http://%s/debug", lookupd.RealHTTPAddr())
+	d := make(map[string]interface{})
+	err = http_api.NegotiateV1("GET", endpoint, &d)
+	data := simplejson.New()
+	data.SetPath(nil, d)
```

Comment:
seems like we could use a little helper for this dance we're doing here a thousand times?